docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Apache Kafka® MirrorMaker 2 log analysis using the offset sync tool
+sidebar_label: Offset sync log analysis
+---
+
+Analyze key log messages during data synchronization with Apache Kafka® MirrorMaker 2 using Aiven’s offset sync inspection tool.
+This tool tracks the progress of synchronization between source and destination clusters.
+You can run it locally after setting up the required environment to review offset
+syncing between source and target Apache Kafka topics. The tool generates messages
+that indicate partition health and highlight when intervention is needed, making it
+particularly useful during large data migrations.
+
+## Prerequisites
+
+Before you download and run the tool, ensure you have the following:
+
+- Basic understanding of Kafka shell tools.
+- A valid Apache Kafka MirrorMaker 2 configuration file with the necessary source and
+  target cluster connectivity fields.
+
+## Download the offset sync inspection tool
+
+Download and run the **offset sync inspection tool** to analyze offset syncing in
+MirrorMaker 2:
+
+- [Download the offset sync inspection tool](https://github.com/aiven/kafka/releases/tag/mm2-offset-sync-inspector-0.1)
+
+## Run the tool
+
+The offset sync inspection tool outputs logs in CSV format. Use the logs to check the
+sync status and detect any issues with the offset syncing process.
+
+### Example command
+
+Run the tool using the following command:
+
+```bash
+<extracted tgz>/bin/kafka-mirrormaker-offset-sync-inspector.sh --mm2-config <path-to-config-file>
+```
+
+Parameter:
+
+- `--mm2-config <path-to-config-file>`: Specifies the path to the MirrorMaker 2
+  configuration file. This file contains the connection details for the source and
+  target Apache Kafka clusters.
+
+To view all available options, run the help command:
+
+```bash
+./bin/kafka-mirrormaker-offset-sync-inspector.sh --help
+```
+
+## Log messages and actions
+
+Below are common log messages generated by the tool, what they mean, and recommended
+actions. If any issues arise, [create a support ticket](/docs/platform/howto/support) or
+email [Aiven Support](mailto:support@aiven.io).
+
+<table>
+  <tr>
+    <th>Message ID</th>
```

Comment:
I think this table has too many columns and doesn't render well on a web page.

Certain things can be combined or removed: The message ID is not part of the script output and it's only in this document to be used in talking about problems between customers and Support. It could be a superscript, or combined with the "Script message" column or whatever.

The Customer Action and Issue with Apache Kafka..." columns could also be combined, or the "Isssue with..." column could be discarded.

Reply:
I’ll streamline the table as suggested.